### ISSUE TITLE
How about a WiKi?

### ISSUE THREAD

**Issue by virgo77** [Rabbit-Company/Passky-Server#18]
Hi,
I'm a new user of this open-source app and I can tell that it's a bit weird to install and use... because there is no real tuto instead the 3 lines on the GitHub page on how to install it with docker. But after that: nothing!

So, I installed it on my local hosted debian VM (as it is asked on the main page) and it now seems to launch on boot. But after that? What should I do? How to connect with the admin account created during install configuration? Are there specific ports to open? (I use UFW to block all incoming traffic but not 80 and 443). But I can't connect however! I tried the app too, but it doesn't connect to my local server because of the user name that must be 6 letters (as 'admin' is only 5...). Great!

So, can someone give more help? Will a tutorial, or maybe a wiki will be available? Because I searched on the web some guide but... nothing! So, ok, it seems to be a great project, but with a good documentation, it will be better for testing purposes!

Thanks a lot

**Comment by zigazajc007** [comment]
Hello, 

My English is really bad, so I'm not the right person to write the documentation, but I would really appriciate it if someone would help me write it. 

Passky Server will by default start on port 8080 (This can be changed in docker-compose.yml file). So you can access Admin Panel on http://serverIP:8080 .Here you can click on "Log In" button and enter your admin creditions. 

Passky Server will assign to port 8080, so you can use proxy manager to install SSL certs and manage domains. The easiest proxy manager to use is "Nginx Proxy Manager". 

Now you can install Passky Client on your device. You can also self-host Passky Client website. 

When you open Passky Client you will see Server field where you can enter url to your Passky Server (in your example: http://serverIP:8080). Note that you can't login with admin creditions here, so you would need to create new account.

**Comment by virgo77** [comment]
Thanks for your reply.

UFW ports opened : 80 // 443 // 8080
I disabled it for testings

When I'm on Firefox or Chrome, the admin page is reachable. But it is very succinct. There's not a lot of options. Maybe it's normal.

So I tried the app, but still no way. The app says that the server is not reachable. I don't know why. So I can't Sign Up.

I add pics where you can see I'm in Sign Up mode:

![Passky - Server Sign Up](https://user-images.githubusercontent.com/23108339/201483871-79676155-9e2e-4382-88ae-6ecaef92aa3a.png)

![Passky - Server Unreachable](https://user-images.githubusercontent.com/23108339/201483786-64f55021-f687-4abf-88ca-76fb36c463f1.png)

**Comment by virgo77** [comment]
Note that the Passky Server is hosted on Debian 11.5 i686

Here are pics about the Admin Page

![Passky - Home Page](https://user-images.githubusercontent.com/23108339/201484155-d31fcb6d-2965-4547-983d-242680133680.png)
![Passky - Server Stats](https://user-images.githubusercontent.com/23108339/201484164-4044af18-4312-44d3-8047-90e0fc1c707a.png)
![Passky - Server Health](https://user-images.githubusercontent.com/23108339/201484179-b62062bf-2105-4eb1-afa9-86d7666f836a.png)
![Passky - Settings](https://user-images.githubusercontent.com/23108339/201484185-ebd0f2a8-f434-4f9f-addf-431937768a6b.png)

**Comment by zigazajc007** [comment]
> So I tried the app, but still no way. The app says that the server is not reachable. I don't know why. So I can't Sign Up.

This can happen if your application only accept secure connection. So it will deny all insecure (http) connections and only allow secure ones (https).

If you have installed Passky from Microsoft Store this can be a reason. So if you want Passky Client to access your Passky Server thru insecure connection (http) I would suggest you to download it manually.

Windows:
- Portable: https://github.com/Rabbit-Company/Passky-Desktop/releases/download/v7.1.0/Passky.7.1.0.exe
- Setup: https://github.com/Rabbit-Company/Passky-Desktop/releases/download/v7.1.0/Passky.Setup.7.1.0.exe

**Comment by virgo77** [comment]
With the portable app, it's working, thanks

I don't remember how I installed the desktop app but it seems to be the Windows Store one, yep.

I will try to install NGiNX Proxy Manager to test SSL connections. I will give feedback here

Thanks for your very helpful replies :)

About documentation, I don't know where you are from, so maybe I can help as I was stuck: my little experience may help for it

**Comment by zigazajc007** [comment]
Awesome, great to hear that.

I'm from Slovenia. I do understand English when I'm reading and also I can type it, but the problem is that my sentences aren't correct (I can't build correct sentences).

**Comment by virgo77** [comment]
yup! As I am too :)
I write as I can say it in my french language.
I don't know when to do it, but maybe I can give you a little doc on how to install and use Passky.

erm, a little request: will Passky be able to use Folders, like in KeePass/XC?
The import functionality works, but from a CSV file it adds correctly all the passwords but they are totally mixed. So I need to search for them as their names are created as they were inside folders, or subfolders.

For example:
Root: HOME
1rst folder: Hypervisor
Sub-Folder: ESXi
Cred.: root (or admin, etc.)

Like this:
HOME
    Hypervisors
         ESXi
             root / password
             other creds
             etc.

So actually, from the import, all the credentials are all mixed. Annoying ^^


I think I will pull a new request for this ^^

See you

**Comment by zigazajc007** [comment]
Good idea, I have added support for folders to my TODO list: https://github.com/orgs/Rabbit-Company/projects/1

**Comment by CMB244** [comment]
sorry, this is just for testing purpose
